js/lykke2.js

```diff
@@ -0,0 +1,1171 @@
+        //                 "blockchainIntegrationType":"none",
+        //                 "blockchainDepositEnabled":false,
+        //                 "isDisabled":false
+        //             }
+        //         ],
+        //         "error":null
+        //     }
+        //
+        const result = {};
+        for (let i = 0; i < currencies.length; i++) {
+            const currency = currencies[i];
+            const id = this.safeString (currency, 'assetId');
+            const code = this.safeString (currency, 'symbol');
+            const name = this.safeString (currency, 'name');
+            const type = this.safeString (currency, 'type');
+            const deposit = this.safeValue (currency, 'blockchainDepositEnabled');
+            const withdraw = this.safeValue (currency, 'blockchainWithdrawal');
+            const active = this.safeValue (currency, 'isDisabled');
+            result[code] = {
+                'id': id,
+                'code': code,
+                'info': currency,
+                'type': type,
+                'name': name,
+                'active': active && deposit && withdraw,
+                'deposit': deposit,
+                'withdraw': withdraw,
+                'fee': undefined,
+                'precision': this.safeNumber (currency, 'accuracy'),
+                'limits': {
+                    'withdraw': {
+                        'min': this.safeValue (currency, 'cashoutMinimalAmount'),
+                        'max': undefined,
+                    },
+                    'amount': {
+                        'min': this.safeValue (currency, 'lowVolumeAmount'),
+                        'max': undefined,
+                    },
+                },
+            };
+        }
+        return result;
+    }
+
+    async fetchMarkets (params = {}) {
+        const response = await this.publicGetAssetpairs (params);
+        const markets = this.safeValue (response, 'payload', []);
+        //
+        //     {
+        //         "payload":[
+        //             {
+        //                 "assetPairId":"AAVEBTC",
+        //                 "baseAssetId":"c9e55548-dae5-44fc-bebd-e72249cb19f3",
+        //                 "quoteAssetId":"BTC",
+        //                 "name":"AAVE/BTC",
+        //                 "priceAccuracy":6,
+        //                 "baseAssetAccuracy":6,
+        //                 "quoteAssetAccuracy":8,
+        //                 "minVolume":0.001,
+        //                 "minOppositeVolume":0.0001
+        //             }
+        //         ],
+        //         "error":null
+        //     }
+        //
+        const result = [];
+        for (let i = 0; i < markets.length; i++) {
+            const market = markets[i];
+            const id = this.safeString (market, 'assetPairId');
+            const name = this.safeString (market, 'name');
+            const [ baseId, quoteId ] = name.split ('/');
```

Comment:
the baseId and quoteId should correspond to the `quoteAssetId` and the `baseAssetId`, the base and quote can be calculated from the `name`